### ISSUE TITLE
Use label for Input title and subtitle

### ISSUE THREAD

**Issue by vozbrann** [Panenco/panenco-ui#280]
There are two ways to pair a label and an input:
```
<!-- IMPLICIT LABEL (not recommended for use) - the label wraps the input. -->
<label> 
  Name:
  <input type="text" name="name" />
</label>

<!-- EXPLICIT LABEL (recommended for use) - the label is connected to an input via "for" and "id" -->
<label for="explicit-label-name">Name: </label>
<input type="text" id="explicit-label-name" name="name" />
```

### Realworld usage
- Sometimes we use only a subtitle.
- "Forgot password" link is part of the subtitle. (Should be refactored. `<label>` should not contain interactive elements)
<img width="450" alt="image" src="https://user-images.githubusercontent.com/39827437/169082533-65d40a33-7bb6-49d5-8a09-61174c254a71.png">

### Multiple labels issue

Multiple `label` elements can be given the same value for their `for` attribute; doing so causes the associated form control (the form control that `for` value references) to have multiple labels.

From the [specification](http://www.w3.org/TR/html4/interact/forms.html#edef-LABEL):
>The LABEL element may be used to attach information to controls. Each LABEL element is associated with exactly one form control.

Creating multiple labels linked to the same input can cause an issue with screen readers.

**Comment by wigqueel** [comment]
Basically, I haven't seen any subtitles in input lables ever. Usually it works like this: there is a (i) button with tooltip of additional info

**Comment by vlacher12** [comment]
@vozbrann What's the status?
